client/src/images/navBooksActive.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="78.141" height="61.022" viewBox="0 0 78.141 61.022"><defs><style>.a{fill:#387c6d}</style></defs><g transform="translate(0.5 0.501)"><path class="a" d="M72.618,2.257C65.279,2.673,50.692,4.19,41.686,9.7a2.059,2.059,0,0,0-.974,1.764V60.2A2.119,2.119,0,0,0,43.831,62c9.265-4.663,22.665-5.936,29.29-6.284a4.177,4.177,0,0,0,4.021-4.106V6.369A4.2,4.2,0,0,0,72.618,2.257ZM35.455,9.7c-9-5.512-23.591-7.027-30.931-7.445A4.2,4.2,0,0,0,0,6.369V51.616a4.175,4.175,0,0,0,4.021,4.106c6.628.348,20.034,1.622,29.3,6.288a2.113,2.113,0,0,0,3.108-1.8V11.442A2.016,2.016,0,0,0,35.455,9.7Z" transform="translate(0 -2.25)"/></g></svg>
```

Comment:
Is this the same svg as above but with other color styles for fill and stroke? If so, you could use an import statement like described [here.](https://create-react-app.dev/docs/adding-images-fonts-and-files/#adding-svgs) and use css to change the values. That way you only need one image.